### DISCUSSION TITLE
Skip hosts when exporting to netbox

### DISCUSSION THREAD

**Discussion by MajaSmereczynska** [kuhn-ruess/cmdbsyncer#29] [Q&A] [answered]
I am using the CMDB Syncer 3.2.3. I am importing hosts from checkmk (2.2) and exporting them to netbox (3.5). Some hosts that I import and are then in the syncer I don't want to export to netbox, I want to skip them. I created a rule in Netbox Custom Attributes to ignore hosts (see below) but it's not working. It would also be fine if they weren't imported from checkmk at all but I didn't see an option for that. 
![image](https://github.com/kuhn-ruess/cmdbsyncer/assets/83644808/9ade8167-886f-49ca-b8f0-094cdcc409ce)
Is there a way to configure it?

**Comment by Bastian-Kuhn** [comment]
Have you clicked "Commit Changes" after?

Rules are cached as long the Host not changes (Means Cache is handled automatically as long you not change rule, then you need to commit).

Then you can check with the ./cmdbsyncer netbox debug_host hostname if your rule matches. Debug Host also deletes the Cache but just for the checked object.

If this not helps, I need to check if there is a bug.

**Reply by MajaSmereczynska** [reply]
I have clicked "Commit Changes" after, but it didn't help. The host haven't been changed either. I debugged the host (see below), but it also didn't help.
![image](https://github.com/kuhn-ruess/cmdbsyncer/assets/83644808/cc09c5af-ad76-4648-9f11-0b8a100bc6ca)

**Reply by Bastian-Kuhn** [reply, marked as answer]
Thanks for the Screenshot, which helped a lot.

I'm unable to test currently, but I just pushed a fix. Can you check if the fix solves the Problem?

**Reply by MajaSmereczynska** [reply]
Yes it does, thank you.
